src/Components/Feelings/Feelings.js

```diff
@@ -1,14 +1,40 @@
-import React from "react";
+import React from 'react';
 import './Feelings.css'
 
-const Feelings = () => {
+const Feelings = ({ setFeeling }) => {
+
+  const callBack = ({ target }) => {
+    setFeeling(target.classList[1]);
+  }
+
   return (
-    <>
-      <h2>How are you Feeling?</h2>
-      <p>Happy</p>
-      <p>Sad</p>
-      <p>Mad</p>
-    </>
+    <section className='feelings-page'>
+      <h1>BEAM</h1>
+      <h2 className='feelings-header'>How are you today?</h2>
+      <section className='feelings-container'>
+        <p className='feeling happy' onClick={callBack}>Happy</p>
```

Comment:
i like how you added a placeholder call back function!

Reply:
agree! But, I'm not sure it's very semantic to name it `callBack`

Reply:
Ah yeah I figured that having one function up top was more readable than having it in every single line 😅

Reply:
@chrissycooper Super fair point - I'm going to adjust and push up the change so when @KatherineBlaine checks it out & merges, that will be fixed!